location.go

```diff
@@ -83,7 +83,7 @@ func init() {
 	timezones = make(map[int]*time.Location, 48)
 	// pre-generate all common timezones
 	for i := -720; i <= 720; i += 30 {
-		logger.Infof("offset: %v", i)
+		logger.Debugf("offset: %v", i)
```

Comment:
If ctx is missing, what will happen?

Reply:
Then there will not be context info in log message. I think it should be fine for some log messages in gosnowflake like this one.